e2e/js_package/.bazelrc

```diff
@@ -0,0 +1,2 @@
+# import common bazelrc shared with e2e workspaces
+import %workspace%/../../.bazelrc.common
```

Comment:
oh that answers my earlier question. maybe a comment over there about where else the bazelrc.common is used